src/main/blackbox/blackbox.c

```diff
@@ -1176,7 +1190,10 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE("Firmware type", "%s",                   "Cleanflight");
         BLACKBOX_PRINT_HEADER_LINE("Firmware revision", "%s %s (%s) %s",    FC_FIRMWARE_NAME, FC_VERSION_STRING, shortGitRevision, targetName);
         BLACKBOX_PRINT_HEADER_LINE("Firmware date", "%s %s",                buildDate, buildTime);
-        BLACKBOX_PRINT_HEADER_LINE("Log start datetime", "%s",              blackboxGetStartDateTime());
+        #ifdef USE_RTC_TIME
+        char buf[FORMATTED_DATE_TIME_BUFSIZE];
+        BLACKBOX_PRINT_HEADER_LINE("Log start datetime", "%s",              blackboxGetStartDateTime(buf));
```

Comment:
Header line should be here even if `USE_RTC_TIME` is not defined. Just use "null" date, ie `0000-01-01T00:00:00`